lms/static/scripts/frontend_apps/components/ContentFrame.tsx

```diff
@@ -17,9 +17,9 @@ export default function ContentFrame({ url, iframeRef }: ContentFrameProps) {
       // too).
       //
       // "autoplay" - Enables Play button to work without first clicking on video
-      // "clipboard-write" - Used by "Copy transcript" button
+      // "clipboard-write *" - Used to copy YouTube transcripts and exported annotations, cross-origin
```

Comment:
Rather than spelling out the individual features that use the permission, which is going to get outdated, it would be easier to spell out which applications need it: Via's video player, the Hypothesis client sidebar.

Reply:
Makes sense